.circleci/config.yml

```diff
@@ -123,7 +123,7 @@ jobs:
 
   prep-build:
     docker:
-      - image: circleci/node:10.17-browsers
+      - image: circleci/node:10.18-browsers
     steps:
       - checkout
       - attach_workspace:
```

Comment:
Below here, `node:10.16` is still being used for `prep-build-test`. That should be updated to `10.18` as well.

It looks like that was my fault actually; I added that job with the wrong node version. Whoops!

Reply:
Good catch, fixed